.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "printWidth": 120,
   "trailingComma": "none",
-  "arrowParens": "avoid"
+  "arrowParens": "avoid",
+  "endOfLine": "auto"
```

Comment:
Let's avoid such changes as they have an effect on the entire repo, plus it's out of the scope and we currently don't want changes to prettier config at the moment 🙏🏼

Reply:
@talkor i understand. But without prettier config I'm getting CRLF errors. Every page I open I see error in every line. :) do you have any idea how can I solve this issue ?